### ISSUE TITLE
Failed to request registration code.

### ISSUE THREAD

**Issue by Nexov25** [WhiskeySockets/Baileys#357]
What method do you want to use? "sms" or "voice" : voice
Failed to request registration code. Please try again.
 {
  email_otp_wait: 0,
  flash_type: 0,
  login: '628577116xxxx',
  reason: 'no_routes',
  retry_after: 3600,
  sms_wait: 2696,
  status: 'fail',
  voice_wait: 0,
  wa_old_wait: 0
}

**Comment by ORMiK0** [comment]
error "no_routes" - try using numbers from other series/country. Try changing the IP/location/language in request

**Comment by Riders004** [comment]
> error "no_routes" - try using numbers from other series/country. Try changing the IP/location/language in request

Same as I got

**Comment by Nexov25** [comment]
> error "no_routes" - try using numbers from other series/country. Try changing the IP/location/language in request

I've tried various ways, but it still doesn't work

**Comment by Riders004** [comment]
> > error "no_routes" - try using numbers from other series/country. Try changing the IP/location/language in request
> 
> I've tried various ways, but it still doesn't work

Same

**Comment by malmeida76** [comment]
same problem

**Comment by xibuyali** [comment]
Regarding registering with whatsapp Business ios, I need someone who can solve it all and I can give you the price you want.

**Comment by lauriolasw** [comment]
same: no_routes

**Comment by xibuyali** [comment]
Yes, if your agreement works, please let me know and I can cooperate

**Comment by Riders004** [comment]
> What method do you want to use? "sms" or "voice" : voice Failed to request registration code. Please try again. { email_otp_wait: 0, flash_type: 0, login: '628577116xxxx', reason: 'no_routes', retry_after: 3600, sms_wait: 2696, status: 'fail', voice_wait: 0, wa_old_wait: 0 }



I'm getting old version reason

**Comment by xibuyali** [comment]
Me too, it returns no routes. Generally, this return means trying again in
an hour.

Riders004 ***@***.***> 于2023年11月21日周二 16:00写道：

**Comment by lairhas** [comment]
Is there any update on this?

**Comment by Riders004** [comment]
Maybe got fixed in latest Baileys

**Comment by lairhas** [comment]
> Maybe got fixed in latest Baileys

I still get this when trying:
```
{
    "reason": "no_routes",
    "status": "fail",
    "retry_after": 3600,
    "sms_wait": 3600,
    "voice_wait": 3600,
    "login": "628777832xxxx",
    "flash_type": 0,
    "wa_old_wait": 3600,
    "email_otp_wait": 3600
}
```

**Comment by Nux-xader** [comment]
same issue, someone know how to use session from android path /data/data/com.whatsapp for whiskeysocket?

**Comment by Riders004** [comment]
> same issue, someone know how to use session from android path /data/data/com.whatsapp for whiskeysocket?

Copy it in your internal storage then upload it to GitHub or some where or copy in your local machine if you are using local machine
